test/cases/duplicate-grid-areas-complex.css

```diff
@@ -0,0 +1,104 @@
+/*******************************\
+  COMPLEX DUPLICATE AREAS TEST 1
+\*******************************/
```

Comment:
Let’s remove this comments. Other cases do not use them.

Reply:
I like those comment blocks. It makes it clear that each bit is a different section testing different functionality.